File: src/main/java/no/nav/syfo/api/selvbetjening/domain/RSGyldighetstidspunkt.java
package no.nav.syfo.api.selvbetjening.domain;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDate;

@Data
@Accessors(fluent = true)
public class RSGyldighetstidspunkt {
    public LocalDate fom;
    public LocalDate tom;
    public LocalDate evalueres;
}

File: src/main/kotlin/no/nav/syfo/api/v3/domain/oppfoelgingsdialog/RSGyldighetstidspunkt.kt
package no.nav.syfo.api.v3.domain.oppfoelgingsdialog

import java.time.LocalDate

data class RSGyldighetstidspunkt(
    var fom: LocalDate? = null,
    var tom: LocalDate? = null,
    var evalueres: LocalDate? = null
)
